app/helpers/patients_helper.rb

```diff
@@ -111,11 +111,23 @@ def household_size_options
   def clinic_options
     clinics = Clinic.all.sort_by(&:name)
     active_clinics = clinics.select(&:active)
-                            .map { |clinic| [clinic.name, clinic.id] }
+                            .map { |clinic| [
+                              clinic.name,
+                              clinic.id,
+                              { data: { naf: !!clinic.accepts_naf, medicaid: !!clinic.accepts_medicaid } }
+                            ]}
                             .unshift nil
-    inactive_clinics = clinics.reject(&:active)
-                              .map { |clinic| [t('patient.abortion_information.clinic_section.not_currently_working_with_fund', fund: FUND, clinic_name: clinic.name), clinic.id] }
-                              .unshift ["--- #{t('patient.abortion_information.clinic_section.inactive_clinics').upcase} ---", nil]
+    inactive_clinics = if clinics.reject(&:active).count > 0
+                         clinics.reject(&:active)
+                           .map { |clinic| [
```

Comment:
is the new `if` check to avoiding calling map on potential `nil`? How would we get a `nil` here, as opposed to just an empty array? If we do worry about `nil`, what do you think about using nil-check invocation `&map` and `&unshift` instead?

Reply:
The `if` check here is to skip this in case there aren't actually any clinics marked inactive. But I do see what you're saying here - let me see if I can simplify.